src/components/PasswordBox.js

```diff
@@ -80,10 +81,13 @@ class PasswordBox extends Nimiq.Observable {
             <div class="prompt nq-text-s" data-i18n="passwordbox-enter-password">Enter your password</div>
             <div password-input></div>
             ${buttonVersions[options.buttonI18nTag]}
-            <svg height="48" width="54" color="inherit" class="loading-spinner">
-                <use xlink:href="#loading-spinner" />
-            </svg>
+            <!-- Loading spinner SVG -->
+            <svg height="48" width="54" color="inherit" class="loading-spinner"><g id="loading-spinner">
```

Comment:
Remove double id `loading-spinner`

Reply:
What do you mean? In this line one is a class and one is an id. Or do you mean that side-wide it's a double ID because of the one on the page already?
It still works as it is, however, because browsers are lenient to such 'errors'.

Reply:
Yes, I meant the duplicate id on the page.
I know that browser can handle such errors.
Still, id's should be unique.

Reply:
I agree with @danimoh. 
I think the id is obsolete anyways since the spinner is no longer referenced by a use tag.